HarmonyCore.Test/EF/MultXfConn.dbl

```diff
@@ -160,6 +161,48 @@ namespace HarmonyCore.Test.EF
 			startServ.Start()
 			startServ.WaitForExit()
         endmethod
+
+.ifdef DEBUG
+        {TestMethod}
+        public method BalancingTest, void
```

Comment:
Can you add a test to handle routing around a long running usage? right now we're pretty much just testing that it works in semi round robin mode